WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -21,6 +18,12 @@ final class CardReaderSettingsConnectedViewController: UIViewController {
     ///
     private var sections = [Section]()
 
+    /// Accept our viewmodel
+    ///
+    func configure(viewModel: CardReaderSettingsPresentedViewModel) {
+        self.viewModel = viewModel as? CardReaderSettingsConnectedViewModel
```

Comment:
Do we really need to downcast? Assuming we do, it might be a good idea to at least DDLog if the downcast fails. You never know...

Reply:
We do, in order to access the conforming model particulars for connected views.